tools/DependencyUpdater/Program.cs

```diff
@@ -850,7 +840,7 @@ internal static string GetLibNameFromRegexMatch(Match match)
             Console.WriteLine($"INFO: adding changes");
 
             // add changes
-            if (!RunGitCli("add -A", workingDirectory))
+            if (!RunGitCli("add --all -- :!*.runsettings", workingDirectory))
```

Comment:
I thought the same, but it's the cleanest way to do it. 

I've considered other alternatives, like:
1. First git -A (puts all files into) and then git -rm "*.runsettings" - this one throws error if there is no .runsettings file
2. Add all files, then reset one by one - too complicated (would require to scan for runsettings)
3. Include into .gitignore - we can't 
4. https://stackoverflow.com/questions/4475457/add-all-files-to-a-commit-except-a-single-file - based on that, I've choose the simplest with just excluding them from "git add"

On  the other hand, why updating nuget package overrides .runsettings file? I think it would behave the same when updating manually?

Reply:
Well... maybe we should replace the nuget option `-FileConflictAction` to `IgnoreAll` and that won't replace the runsettings file anymore, Care to test?

Reply:
Well, It would help if the .runsettings file does exists. When there is none, the new one will be created and add into commit. Keep in mind, that with latest changes in VS there can be one .runsettings file at the "sln" level.

![image](https://user-images.githubusercontent.com/19893965/205311354-5229720e-ab3e-4b17-b21b-fc2aa734a8b6.png)

BTW Is the "nano.runsettings" name valid? Shouldn't it be ".runsettings"?

Reply:
OK. Thanks. I guess we'll have to take this as the reasonable solution for this. Please go ahead. Just add a verbose comment there explaining why the need to ditch any runsettings settings file being added as result of the update.

Reply:
Currently there is no way (at least I couldn't find one) for the Unit Test project template to add a global runsettings to the solution. Only inside the project it's creating. And yes, the template should have a `.runsettings` file instead of a nano one.

Reply:
I've change foreach update look, now it's iterating through each lib and then prints output. Also, if there were any errors it will exit with code "1".

Reply:
Added comment